chart/templates/post-delete-hook-cluster-role.yaml

```diff
@@ -12,20 +12,31 @@ rules:
   - apiGroups: [ "extensions","apps" ]
     resources: [ "deployments" ]
     verbs: [ "get", "list", "delete" ]
+  - apiGroups: [ "batch" ]
+    resources: [ "jobs" ]
+    verbs: [ "get", "list", "watch", "delete", "create" ]
```

Comment:
All these permissions weren't previously necessary, and are only necessary because of the new job ?

Reply:
yes, it is because of the new job in rancher-webhook.
Basically, what happens is that rancher-post-delete-hook uses its own serviceAccount to run the `helm uninstall rancher-webhook` command in its post-delete-job pod, and rancher-webhook has a pre-delete-hook which creates Job, SA, ClusterRole, ClusterRoleBinding and etc., so we need to give rancher-post-delete-hook those permissions.